### DISCUSSION TITLE
有没有办法一个配置文件支持多种评测任务

### DISCUSSION THREAD

**Discussion by NickLennonLiu** [open-compass/opencompass#181] [View all discussions]
> 在评测阶段，我们一般以数据集本身的特性来选取对应的评估策略，最主要的依据为**标准答案的类型**，一般以下几种类型：
> - **选项**：常见于分类任务，判断题以及选择题，目前这类问题的数据集占比最大，有 MMLU, CEval数据集等等，评估标准一般使用准确率--`ACCEvaluator`。
> - **短语**：常见于问答以及阅读理解任务，这类数据集主要包括CLUE_CMRC, CLUE_DRCD, DROP数据集等等，评估标准一般使用匹配率--`EMEvaluator`。
> ...

我现在希望在一次评测任务中同时测试模型在多个场景任务下的能力，有办法实现吗？

**Comment by Ezra-Yu** [comment]
> 我现在希望在一次评测任务中同时测试模型在多个场景任务下的能力，有办法实现吗？

可以解释一下‘多个场景任务‘的意思吗？举个例子
